src/main/java/DBAccesser/User/QueryUser.java

```diff
@@ -0,0 +1,97 @@
+                                .bind("EmailID")
+                                .to(emailID)
+                                .build();
+        List<User> resultSet = spannerTemplate.query(User.class, statement, new SpannerQueryOptions().setAllowPartialRead(true)); //setAllowPartialRead for reading specific columns
+ 
+        return (resultSet.size() > 0);
+    }
+
+    public boolean checkIfUserIDExists(long userID) {
+
+        String SQLStatment = "SELECT UserID FROM User WHERE UserID=@userID";
+        Statement statement = Statement.newBuilder(SQLStatment)
+                                .bind("UserID")
+                                .to(userID)
+                                .build();
+        List<User> resultSet = spannerTemplate.query(User.class, statement,  new SpannerQueryOptions().setAllowPartialRead(true));
+ 
+        return (resultSet.size() > 0);
+    }
+
+    public long getUserIDFromUsername(String username) {
+
+        String SQLStatment = "SELECT UserID FROM User WHERE Username=@Username";
+        Statement statement = Statement.newBuilder(SQLStatment)
+                                .bind("Username")
+                                .to(username)
+                                .build();
+        List<User> resultSet = spannerTemplate.query(User.class, statement,  new SpannerQueryOptions().setAllowPartialRead(true));
+ 
+        return resultSet.get(0).getUserID();
+    }
+
+    public long login(String username, String password) {
+        String SQLStatment = "SELECT UserID from User WHERE Username=@username and Password=@password";
+        Statement statement = Statement.newBuilder(SQLStatment)
+                                .bind("username")
+                                .to(username)
+                                .bind("password")
+                                .to(password)
+                                .build();
+        List<User> resultSet = spannerTemplate.query(User.class, statement,  new SpannerQueryOptions().setAllowPartialRead(true));
+        if(resultSet.size() == 0) {
+            return -1;
+        }
+        return resultSet.get(0).getUserID();
+    }
+
+    public Map<String, Object> getUser(String username) {
+        String SQLStatment = "SELECT UserID, Username, EmailID, MobileNo, Picture FROM User WHERE Username=@Username";
+        Statement statement = Statement.newBuilder(SQLStatment)
+                                .bind("Username")
+                                .to(username)
+                                .build();
+        List<User> resultSet = spannerTemplate.query(User.class, statement,  new SpannerQueryOptions().setAllowPartialRead(true));
+        Map<String, Object> response = new HashMap<String, Object>();
+        if(resultSet.size() == 0) {
+            response.put("userID", -1);
+            return response;
+        }
+        User user = resultSet.get(0);
+        response.put("userID", user.getUserID());
+        response.put("username", username);
```

Comment:
Should write the keys as `UserID` and `Username` to make it consistent with other keys here (`EmailID`, `MobileNo`) and all responses in #3